Playbooks/playbook-ArcSight_Test.yml

```diff
@@ -0,0 +1,468 @@
+    type: regular
+    task:
+      id: fc13a703-fd60-4d46-84c1-3db0896c84bb
+      version: -1
+      name: add entries
+      description: ""
+      script: '|||as-add-entries'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "13"
+    scriptarguments:
+      entries:
+        simple: '[{"name": "t3", "EventID": "9"},{"name": "t4", "EventID": "9"}]'
+      resourceId:
+        simple: H0LvlmWgBABCA5+HbRyHZoQ==
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 1915
+        }
+      }
+    note: false
+    timertriggers: []
+  "12":
+    id: "12"
+    taskid: 340d21e5-303f-409a-8e2d-17b2adbd1850
+    type: regular
+    task:
+      id: 340d21e5-303f-409a-8e2d-17b2adbd1850
+      version: -1
+      name: get all query viewers
+      description: ""
+      script: '|||as-get-all-query-viewers'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "14"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 2265
+        }
+      }
+    note: false
+    timertriggers: []
+  "13":
+    id: "13"
+    taskid: c15d4bc1-6af7-4886-83ed-0c311d0a5c89
+    type: regular
+    task:
+      id: c15d4bc1-6af7-4886-83ed-0c311d0a5c89
+      version: -1
+      name: clear entries
+      description: ""
+      script: '|||as-clear-entries'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "12"
+    scriptarguments:
+      resourceId:
+        simple: H0LvlmWgBABCA5+HbRyHZoQ==
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 2090
+        }
+      }
+    note: false
+    timertriggers: []
+  "14":
+    id: "14"
+    taskid: 47f51ce9-b55c-44f6-82f4-335f61d31c31
+    type: title
+    task:
+      id: 47f51ce9-b55c-44f6-82f4-335f61d31c31
+      version: -1
+      description: ""
+      name: done
+      type: title
+      iscommand: false
+      brand: ""
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 2440
+        }
+      }
+    note: false
+    timertriggers: []
+  "15":
+    id: "15"
+    taskid: 34eb1a66-9283-44bb-843c-ea13c22e54b0
+    type: title
+    task:
+      id: 34eb1a66-9283-44bb-843c-ea13c22e54b0
+      version: -1
+      name: good luck
+      description: ""
+      type: title
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "4"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 195
+        }
+      }
+    note: false
+    timertriggers: []
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 2455,
+        "width": 380,
+        "x": 50,
+        "y": 50
+      }
+    }
+  }
+inputs: []
+outputs: []
+tests:
```

Comment:
remove the tests - it is a test playbook

Reply:
You've failed to provide tests for:
ArcSight ESM Test

Reply:
it is because your test playbook is not in TestPlaybooks folder. change the folder